### PR TITLE
SLM-79 pdfs

### PR DESCRIPTION
**Pull request by mikehalmamoj**
* give pdfs a generic filename
* prevent update of recipients after barcode documents are generated
* remove autofocus unless essential, it's a bit rubbish

### CODE REVIEW

**Review by smcveigh941 — DISMISSED** [review]
LGTM

File: assets/js/page-enhancements.js [hunk 1]
```diff
@@ -80,33 +80,8 @@
       })
   })
 })($, document)
-;(autoDownloadCoversheetPdf = ($, document) => {
```
Comment (nathanrussell-moj-digital) [inline assets/js/page-enhancements.js:83]: Do we not want to auto download anymore?

Comment (mikehalmamoj) [reply assets/js/page-enhancements.js:83]: Whoops!

Comment (smcveigh941) [reply assets/js/page-enhancements.js]: 👨🏻‍🦯

File: integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts [hunk 2]
```diff
@@ -14,13 +13,14 @@ context('Legal Sender Journey E2E', () => {
...
-  })
 
-  it('should allow Legal Senders to perform all actions as part of their workflow', () => {
     // Request a magic link and click it to sign in and land on the Find Recipient By Prison Number page
     cy.visit('/link/request-link')
     Page.verifyOnPage(RequestLinkPage).submitFormWithValidEmailAddress('valid@email.address.cjsm.net')
     cy.visit('/link/verify-link?secret=a-valid-secret')
+  })
+
+  it('should allow Legal Senders to perform all actions as part of their workflow - images', () => {
```
Comment (nathanrussell-moj-digital) [inline integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts:23]: Do we want an E2E that does all of this, but is for the PDF coversheet journey?

Comment (mikehalmamoj) [reply integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts:23]: I did that at first. But it seemed a waste of time as it just repeats all the same steps. The PDF covesheet journey is also tested elsewhere - hence I left it like this.

Comment (nathanrussell-moj-digital) [reply integration_tests/integration/e2e/legalSenderJourneyE2e.spec.ts:23]: ok 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]

**Pull request merged**